packages/shared/package.json

```diff
@@ -19,8 +19,8 @@
     "lint": "eslint ./",
     "clean": "shx rm -rf ./dist",
     "type": "tsc --project ./tsconfig.json --declaration --emitDeclarationOnly",
-    "build:esm": "esbuild src/index.ts --bundle --sourcemap --target=es2016 --format=esm --outfile=./dist/esm/index.js --analyze",
-    "build:cjs": "esbuild src/index.ts --bundle --sourcemap --target=es2016 --format=cjs --outfile=./dist/cjs/index.js --analyze",
+    "build:esm": "esbuild src/index.ts --bundle --external:events --sourcemap --target=es2016 --format=esm --outfile=./dist/esm/index.js --analyze",
```

Comment:
We do this same thing already in web, except it's `--external:@openfeature/core`, which prevent us from bundling the core module there.